content/maxwell1_fundamentals/dipole_sources_in_homogeneous_media/index.rst

```diff
@@ -3,37 +3,93 @@
+    	- Asymptotic expressions are provided for several cases.
+    	- Numerical modeling tools are made available for investigating the dependency of the electric and magnetic fields on various parameters.
 
 Dipole sources are fundamental electromagnetic sources which exist at a single point in space.
-In geophysics, there are generally two types of dipole sources: electrical current dipole sources and magnetic dipole sources.
-Although true dipole sources do not exist in nature, they do very well at approximating the electromagnetic sources used in many geophysical applications.
-On this page, we will focus on defining the term dipole source and show how dipole sources can be represented within Maxwell's equations.
+Although true dipole sources do not exist in nature, they do very well at approximating the electromagnetic sources used for many geophysical applications.
+In geophysics, there are two types of dipole sources: electrical current dipole sources and magnetic dipole sources.
+
+In matter, the electrical current dipole (:math:`\mathbf{p}`) generates a primary current density in the surrounding region; free current does not flow in free-space.
+This is illustrated in :numref:`DipolesFig` (left).
+A magnetic dipole, on the other hand, generates a primary magnetic field in the surrounding region.
+This is illustrated in :numref:`DipolesFig` (right).
+Notice how the field lines converge at a single point in space for either source.
+We will discuss the secondary electric and magnetic fields generated by each dipole in the subsequent material.
+
+
+.. figure:: images/dipoles.png
+		:align: center
+		:figwidth: 100%
+		:name: DipolesFig
+		
+		(Left) Electrical current dipole ( :math:`\mathbf{p}`) oriented in the :math:`\hat x` direction and the primary current density ( :math:`\mathbf{J}`) it produces. (Right) Magnetic dipole ( :math:`\mathbf{m}`) oriented in the :math:`\hat y` direction and the primary magnetic field ( :math:`\mathbf{H}`) it produces.
+
+
+Both electrical current dipoles and magnetic dipoles can be represented as source terms in Maxwell's equations.
+For the electrical current dipole, the source term (:math:`\mathbf{J_e^s}`) represents an electrical current density and has units A/m :math:`\!^2`.
+For the magnetic dipole, the source results from a magnetization (:math:`\mathbf{M}`).
+However, the magetic source term (:math:`\mathbf{J_m^s} = - i\omega \mu \mathbf{M}`) is generally represented by a magnetic current density, where :math:`\mathbf{J_m^s}` has units V/m :math:`\!^2`.
+Details regarding the electric and magnetic current densities were provided along with the :ref:`Ampere-Maxwell equation<ampere_maxwell>`.
+
+In the presence of an electromagnetic source, Maxwell's equations in the frequency domain are given by:
+
+.. math::
+	\begin{split}
+	\nabla \times \mathbf{E} + i \omega & \mu \mathbf{H} = \pm \, \mathbf{J_m^s}  \\
```

Comment:
Depends on the convention you want to use. In Ward and Hohmann they added a negative sign for some things because it made the math cleaner. I stated explicitly that the \pm depends on the sign convention.